src/components/Semester.vue

```diff
@@ -152,9 +164,7 @@ export default {
       return allWarnings;
     },
     concurrentSubjects: function () {
-      return this.selectedSubjects.filter(subj => {
-        return subj.semester <= this.index;
-      });
+      return this.selectedSubjects.slice(0, this.index + 1).flat();
```

Comment:
Hmm, not sure we should just forget Microsoft especially since a lot of the places it's used would break it.

Reply:
So you think we should just flatten it manually? I guess that's fine. It should definitely get its own function in that case though.

Reply:
Does anyone have internet explorer to test? There could be other unsupported things too and if so maybe it should be wrapped into one issue for brower support.  Otherwise I can make a flattening mixin

Reply:
Just tested on Edge and dragging throws errors (on the main site). So I agree. Flatten for now, Browser support later.